docs/examples.md

```diff
@@ -626,11 +626,13 @@ Sometimes you may want to create a class programmatically.
 >>> from attrs import make_class
 >>> @define
 ... class C1:
-...     x = field()
+...     x = field(type=int)
```

Comment:
this broke the test in 632; technically it should pass if you add an `: int` annotation to `y`?

Reply:
I think I fixed it. Is this ok?